core/frontend/services/redirects/settings.js

```diff
@@ -117,7 +117,7 @@ const getBackupRedirectsFilePath = (filePath) => {
     return path.join(dir, `${name}-${moment().format('YYYY-MM-DD-HH-mm-ss')}${ext}`);
 };
 
-const setFromFilePath = (filePath, ext) => {
+const setFromFilePath = (filePath, ext = '.json') => {
```

Comment:
Adding a comment about why this default is here would be good. Specifically that API v2 used to support only `.json` extension and this default should go away with `Ghost v4` :)

Reply:
I've left a comment about that.